src/page-content/navigation/navigation-header.tsx

```diff
@@ -114,8 +114,8 @@ export const dropdownMenu: Pick<NavProps, 'dropdownMenu'> = {
     },
     {
       title: (
-        <Button to={'/jobs'} style={'link'}>
-          Jobs
+        <Button to={'/summit21/'} style={'link'}>
```

Comment:
Why the trailing slash
```suggestion
        <Button to={'/summit21'} style={'link'}>
```